lib/oli_web/templates/pow/session/author_new.html.eex

```diff
@@ -14,10 +14,6 @@
       <%= error_tag f, :password %>
     </div>
     <div class="mb-4 d-flex flex-row">
-      <div class="custom-control custom-checkbox flex-grow-1">
```

Comment:
We should leave this checkbox here for users who may be accessing from a shared/non-secure browser and default to not setting a persistent session